contracts/src/Teleporter/upgrades/TeleporterRegistry.sol

```diff
@@ -5,28 +5,109 @@
 
 pragma solidity 0.8.18;
 
-import "../../WarpProtocolRegistry.sol";
 import "../ITeleporterMessenger.sol";
+import "@subnet-evm-contracts/interfaces/IWarpMessenger.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
```

Comment:
Per the OZ audit, let's explicity import the types of interest rather than the entire contract